src/App.tsx

```diff
@@ -1,26 +1,84 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import { useState } from 'react';
+import cn from 'classnames';
 
-const USER_ID = 0;
+import './styles/trasitiongroup.scss';
+import { TransitionGroup, CSSTransition } from 'react-transition-group';
 
-export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+import { getFilteredTodos } from './utils/getFilteredTodos';
+
+import { useTodosContext } from './context/TodosContext';
+
+import { ErrorsContainer } from './components/ErrorsContainer/ErrorsContainer';
+import { TodoItem } from './components/TodoItem/TodoItem';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer/Footer';
+
+import { Status } from './types/Status';
+import { deleteTodo } from './api/todos';
+
+export const App = () => {
+  const {
+    todos,
+    errorMessage,
+    tempTodo,
+    setLoadingIds,
+    setTodos,
+    setErrorMessage,
+  } = useTodosContext();
+  const [status, setStatus] = useState<Status>(Status.All);
+
+  const filteredTodos = getFilteredTodos(todos, status);
```

Comment:
This logic may also go to your context

The responsibility should be the following:
Context rules all data
Components decide how this data is displayed